editor/enable-tracking-prompt/test/index.js

```diff
@@ -12,35 +13,37 @@ import {
 } from '../';
 
 describe( 'EnableTrackingPrompt', () => {
-	const tracking = require( '../../utils/tracking' ); // no default export
 	const originalSetUserSetting = window.setUserSetting;
-	const originalBumpStat = tracking.bumpStat;
+	const originalDocumentAddEventListener = document.addEventListener;
+	let eventMap = {};
 	let removeNotice;
 
 	beforeEach( () => {
 		window.setUserSetting = jest.fn();
-		tracking.bumpStat = jest.fn();
+		document.addEventListener = jest.fn( ( event, cb ) => {
+			eventMap[ event ] = cb;
+		} );
```

Comment:
This approach is from https://github.com/airbnb/enzyme/issues/426#issuecomment-228601631.